metadata/gorefs.schema.yaml

```diff
@@ -0,0 +1,43 @@
+type: map
+mapping:
+  "layout":
+    type: str
+    required: true
+    enum: [goref]
+  "id":
+    type: str
+    required: true
+    pattern: /GO_REF:[0-9]{7}/
+  "authors":
+    type: str
+    required: true
+  "year":
+    type: number
+    required: false
+  "external_accession":
+    type: seq
+    required: false
+    sequence:
+      - type: str
+        required: false
+        pattern: /[a-zA-Z\_]+:[0-9a-zA-Z\_]+/
+  "alt_id":
+    type: seq
+    required: false
+    sequence:
+      - type: str
+        required: false
+        pattern: /GO_REF:[0-9]{7}/
```

Comment:
Doe alt_id only get to be a GO_REF ID?

Reply:
Apparently there are no counterexamples/